simulator/src/api/marketapi.ts

```diff
@@ -49,4 +49,60 @@ router.get('/suggested-price', authenticate('manager'),
 });
 
 
+/**
+ * Get the market.
+ */
+router.get('/', authenticate(), 
+    (req: express.Request, res: express.Response) => {
+    if (req.actor == undefined) return res.send(401).send("Not authenticated!");
+    try {
+        let state = Simulation.getState();
+        if (req.actor.role == 'prosumer') {
+            const market =  state.houses[req.actor.id].powerPlant?.market;
+            if (market != undefined) {
+                return res.status(200).json(market);
+            } else {
+                return res.status(400).send("No market found");
+            }
+        } else if (req.actor.role == 'manager') {
+            return res.status(200).json(state.powerPlants[req.actor.id].market);
+        } else {
+            return res.status(400).send("Permission denied! Only accessable by prosumers and managers.");
+        }
+    } catch(err) {
+        console.trace(err);
+        console.log("[MarketAPI] Failed to find a requested power plant with id " + req.actor.id + ".");
+    }
+    return res.status(400).send("Whoops! We failed to find requested house, please try again later.");
+});
+
+
+/**
+ * Get the market.
```

Comment:
```suggestion
 * Get the current market price.
```

Reply:
The price getter will either pick the suggested price or the currently set price, _price can be undefined and then suggestedPrice is choosen instead